### PR TITLE
Fix bug with .fill bytes > 127 from other encoding

### PR DESCRIPTION
**Pull request by emilbayes**

### CODE REVIEW

**Comment by dcousens** [comment]
LGTM,  but not sure why it was that way before

**Comment by emilbayes** [comment]
@dcousens Node.js delegates `.fill` to a native method. Maybe it was a brain fart? As far as I can see there are no tests covering bytes > 127, but I did't read all the Node tests either so my knowledge is only superficial

File: test/base64.js [hunk 1]
```diff
@@ -44,3 +44,12 @@ test('base64: invalid non-alphanumeric characters -- should be stripped', functi
   )
   t.end()
 })
+
+test('base64: high byte', function (t) {
+  var highByte = Buffer.from([128])
```
Comment (feross) [inline test/base64.js]: This should be `B.from`, right? None of the other tests reference `Buffer`.

Comment (emilbayes) [inline test/base64.js]: Oh woops

**Pull request merged**